src/main/java/com/denizenscript/depenizen/bukkit/bridges/MythicMobsBridge.java

```diff
@@ -161,6 +285,14 @@ public static MythicMob getMythicMob(String name) {
         return MythicBukkit.inst().getMobManager().getMythicMob(name).orElse(null);
     }
 
+    public static Map<String, Double> getDamageModifiers(MythicMob mob) {
+        return mob.getDamageModifiers();
+    }
+
+    public static Collection<String> getMobNames() {
+        return getMobManager().getMobNames();
```

Comment:
Same here - can probably use `getMobManager().getMobNames()` directly, it doesn't significantly shorten the code / make it cleaner